cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/prometheus/PrometheusMetricSampler.java

```diff
@@ -105,13 +106,15 @@ private void configurePrometheusAdapter(Map<String, ?> configs) {
                 "%s config is required by Prometheus metric sampler", PROMETHEUS_SERVER_ENDPOINT_CONFIG));
         }
 
+        final String path = (String) configs.get(PROMETHEUS_SERVER_PATH_CONFIG);
```

Comment:
Just curious won't this cause NPE if PROMETHEUS_SERVER_PATH_CONFIG is not set?

Reply:
In theory, yes, but this is set a few lines up: https://github.com/linkedin/cruise-control/pull/2082/files/aeb29411e60ab15884c0f7696cc639ef81d00246#diff-d114eb44a5084d7894e3ddc1b1e00f52466bfd49f8c3b40383b8863534683c0fR55 and follows the existing pattern used for the Prometheus server endpoint.